src/table/townname.h

```diff
@@ -650,14 +650,14 @@ static const char * const _name_spanish_real[] = {
 	"Elorza",
 	"Arismendi",
 	"Trujillo",
-	"Carupano",
+	"Car\xC3\xBA""pano",
```

Comment:
The double "" looks weird to me. I can't think that it's necessary, but I can see others like that that you haven't changed either. Maybe fix those while you're at it?

Reply:
The reason I did this is because I received errors like this before:
```
In file included from /home/nik/src/OpenTTD/src/saveload/oldloader_sl.cpp:37:
/home/nik/src/OpenTTD/src/saveload/../table/townname.h:687:2: warning: hex escape sequence out of range
  "Yacu\xC3\xADba",
  ^~~~~~~~~~~~~~~~
[SRC] Compiling townname.cpp
In file included from /home/nik/src/OpenTTD/src/townname.cpp:21:
/home/nik/src/OpenTTD/src/table/townname.h:687:2: warning: hex escape sequence out of range
  "Yacu\xC3\xADba",
```

Which, I'm assuming is why this `""` concatenation is present in a few other place in this file. It looks like in this case though, I can do `"Car\xC3\xBApano"` without getting this compiler warning with gcc 8.2.0.

Reply:
ah...so it's a terminator to the hex sequence?

Reply:
Yeah, it seems to be a workaround to terminate the hex sequence. Just ends the string and concatenate with a new one.

I'm rusty at C so just learning as I go >_<

Reply:
Anyways, I've removed the `""` on Carúpano since I don't get a warning without it, but Yacuíba still has the double-quote, since otherwise I get a warning.